test/remote/gateways/remote_blue_snap_test.rb

```diff
@@ -292,7 +292,7 @@ def test_successful_purchase_with_currency
   end
 
   def test_successful_purchase_with_level3_data
-    l_three_visa = credit_card('4111111111111111', month: 2, year: 2023)
+    l_three_visa = credit_card('4111111111111111', month: 2, year: Time.now.year + 1)
```

Comment:
Why do we need to provide month and year here?

Reply:
I'm not sure about the month, sometimes sandboxes have different months passed in for different secenarios, but I can remove the year since I just updated it so the test wouldn't fail for expiration date invalid.

Reply:
I removed month and year everywhere in this file and it doesn't affect the testing outcomes!